scrapy/__init__.py

```diff
@@ -35,5 +35,6 @@
 from scrapy.http import Request, FormRequest
 from scrapy.selector import Selector
 from scrapy.item import Item, Field
+from scrapy.pipelines import ItemPipeline
```

Comment:
What tests? 
I did this so you can `from scrapy import ItemPipeline`. Like you can with all other major public API components, like `Spider`, `Selector`, `Item` etc.. 

It might no be fully pep8 compliant, but according to the scrapy devs, consistency is more important, and this is in favor of the user so.....